addspecdlg.py

```diff
@@ -16,19 +16,19 @@
 
 """
 
-from PyQt4.QtCore import *
-from PyQt4.QtGui import *
-from ui.xga import ui_MessageDlg
+from PyQt6.QtGui import *
+from PyQt6.QtWidgets import QDialog
+from ui import ui_MessageDlg
 
 
 class addspecedlg(QDialog, ui_MessageDlg.Ui_messageDlg):
     def __init__(self,  parent=None):
         super(addspecedlg, self).__init__(parent)
         self.setupUi(self)
         self.msgLabel.palette().setColor(self.msgLabel.backgroundRole(), QColor(255, 255, 255))
```

Comment:
I have discovered that setting  the background for labels like this in fails silently in Qt6. I'm sure there is a way to do this programmatically in Qt6, but I have been setting the background color for these labels in the .ui file using QtDesigner.